core/src/instance.rs

```diff
@@ -197,7 +197,7 @@ impl Instance {
             *state = new_state;
         }
 
-        // context.log(format!("trace/reduce: {:?}", action_wrapper.action()));
+        context.log(format!("trace/reduce: {:?}", action_wrapper.action()));
```

Comment:
Are you playing tag on this line of code? Didn't we uncomment it once now it's recommented and now you're uncommenting it again?

Reply:
I added an "exclude: `^trace/`" rule to LogRules::default, which will allow this line to stay in, what do you think @Connoropolous @zippy ?

Reply:
Yes, approved.